src/styles/blocks/container.scss

```diff
@@ -0,0 +1,63 @@
+.container {
+  width: #{$container-size};
+  margin: auto;
+  padding: 10px;
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$gap-size};
+
+  @for $index from 28 through 31 {
+    &--month-length--#{&index}
+    &__cell::nth-child(n + #{index} + 1) {
+      display: none;
+    }
+  }
+
+  $days-of-week:
+  "mon" 0,
+  "tue" ($cell-size + $gap-size),
+  "wed" (($cell-size + $gap-size) * 2),
+  "thu" (($cell-size + $gap-size) * 3),
+  "fri" (($cell-size + $gap-size) * 4),
+  "sat" (($cell-size + $gap-size) * 5),
+  "sun" (($cell-size + $gap-size) * 6);
+
+  @each $day, $day-shift in $days-of-week {
+    &--start-day--#{$day} 
+    &__day:first-of-type {
+      margin-left: #{$day-shift};
+    }
+  }
+
+  &__cell {
+    box-sizing: border-box;
+    background-color: #{$main-color};
+    border: 1px solid #{$border-color};
+  
+    font-family: "Arial", sans-serif;
+    font-size: 30px;
```

Comment:
you can create normalize.scss and transfer your font-family and font-size
also, you can add your "box-sizing: border-box" on all elements * in normalize.scss

but this is only my opinion that it will be better

Reply:
agree, you should do it